src/Exception/DispatchingException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Equip\Exception;
+
+use InvalidArgumentException;
+
+class DispatchingException extends InvalidArgumentException
+{
+    /**
+     * @param string $spec
+     *
+     * @return static
+     */
+    public static function notInvokable($spec)
+    {
+        return new static(sprintf(
+            'Dispatcher `%s` is not invokable',
+            $spec
```

Comment:
But the exception would never happen in that scenario, because it will pass the `is_callable` check.